src/streams.h

```diff
@@ -278,7 +278,7 @@ class CDataStream
             vch.insert(it, first, last);
     }
 
-    void insert(iterator it, const char* first, const char* last)
+    void insert(iterator it, const value_type* first, const value_type* last)
```

Comment:
I'm slightly surprised that this doesn't give an error anywhere. `value_type` was `uint8_t` before, not `char`.

Reply:
This interface is not used in our code. It was added to have full compat with the std::vector interface. No opinion on whether that is a good goal.

Reply:
I think that's a reasonably valid goal. Though we should at least test the function. Not in this PR though.